Azure-Samples/modern-data-warehouse-dataops
# Single tech/databricks

# Type of PR

- Code changes

## Purpose

- Fix bug #253. By removing service enpoints from other subnets other than firewall subnet

## Author pre-publish checklist


## Validation steps

- Deploy sample2
  ![image](https://user-images.githubusercontent.com/18494471/116854387-f1d0b680-ac3a-11eb-9154-7991a5466972.png)

  ![image](https://user-images.githubusercontent.com/18494471/116851622-0f4f5180-ac36-11eb-90ec-a206788839b1.png)

- Deploy sample3
  ![image](https://user-images.githubusercontent.com/18494471/116851681-2d1cb680-ac36-11eb-9094-c10c0e3b802c.png)

- Run sample3 notebooks
  ![image](https://user-images.githubusercontent.com/18494471/116854234-b635ec80-ac3a-11eb-9dd1-b1acaeca3ae2.png)
  
  ![image](https://user-images.githubusercontent.com/18494471/116854514-217fbe80-ac3b-11eb-8602-ae46f95b4b30.png)

- Try to access any blob from databricks cluster and it should be blocked
  ![image](https://user-images.githubusercontent.com/18494471/116854650-5e4bb580-ac3b-11eb-9f32-7c83e5cbb5c6.png)
  
  ![image](https://user-images.githubusercontent.com/18494471/116854756-8dfabd80-ac3b-11eb-9a73-de9fadc07d86.png)
  
  ![image](https://user-images.githubusercontent.com/18494471/116854876-b97da800-ac3b-11eb-8626-bbebe2ce2ad3.png)


## Issues Closed or Referenced

- Closes #253
